src/cromshell/utilities/io_utils.py

```diff
@@ -42,3 +53,35 @@ def doomed_logo():
     print(colored(r"=='    _-'                                                            \/   `==", "red"))
     print(colored(r"\   _-'                                                                `-_   /", "red"))
     print(colored(r".`''                                                                      ``'.", "red"))
+
+
+def assert_required_file_is_not_empty(file_name: str, file_description: str):
+    """Confirm the provided file exist and is not empty."""
+
+    if file_name is None:
+        LOGGER.info("%s was not provided", file_description)
+        return
+    if not Path(file_name).exists():
+        LOGGER.error("ERROR: %s does not exist: %s", file_description, file_name)
+        raise Exception("ERROR: %s does not exist: %s", file_description, file_name)
+    else:
+        if os.path.getsize(file_name) == 0:
+            LOGGER.error("ERROR: %s is empty: %s.", file_description, file_name)
+            raise Exception("ERROR: %s is empty: %s.", file_description, file_name)
+
+
+def validate_workflow_id(workflow_id: str):
+    """Validates a workflow id"""
+
+    pattern = re.compile("[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}")
+    ret_val = 0
+    if not pattern.match(workflow_id):
+        ret_val = 1
+
+    return ret_val
+
+def pretty_print_json(json_text: str):
+    """Prints JSON String in a fancy way"""
```

Comment:
is `json_text` supposed to be a path to a file, or raw JSON text?

Reply:
Not the file path, but the content of the file in a string format

Reply:
I'll add details about input in the function comments